airbyte-integrations/connectors/source-mixpanel/source_mixpanel/components.py

```diff
@@ -261,6 +259,8 @@ class EngagePaginationStrategy(PageIncrement):
     page - incremental page number
     """
 
+    _total = 0
```

Comment:
shouldn't this be reset from the `reset` method since this is part of the inner state?

Reply:
I have added reset function. But it is not directly required